README.md

```diff
@@ -136,24 +170,26 @@ The `visually-hidden` css declaration can be seen below:
 }
 ```
 
-### aria-label
+Why not just use `display: none;`? If you were to do so the element would have it's display turned off. This will make the element and all it's descendants inaccessible to screen readers.
```

Comment:
```suggestion
Why not just use `display: none;`? If you were to do so the element would have it's display turned off. This will make the element and all it's descendants invisible and inaccessible to screen readers.
```